runtimes/pythonrt/py-sdk/autokitteh/__init__.py

```diff
@@ -7,6 +7,10 @@
 from . import errors
 
 
+def nop():
```

Comment:
The `py-sdk` is for external users, not for testing. This should not be here.
You can use `check_connection_name` or another function from `connections` or another function like that.

Reply:
done